openfisca_us/tests/policy/baseline/hhs/medicaid/pregnancy_medicaid_eligible.yaml

```diff
@@ -0,0 +1,67 @@
+- name: Not pregnant 
+  period: 2022
+  input:
+    is_pregnant: False
+  output:
+    pregnancy_medicaid_eligible: False
+- name: Family of 3 eligible for Medicaid but not Pregnancy Medicaid, income at 135% of FPL
```

Comment:
Nice integration tests, but I think we can convert them into unit tests in this file by removing person-level `market_income` and adding spm_unit-level `medicaid_gross_income`. Integration tests can go in integration.yaml.